### PR TITLE
Spectra matching single row

### PR DESCRIPTION
**Pull request by Annexhc**
Hi Tomas,
I have added single row identification as requested in #616. If a single row is selected the results will be added to the feature list row and are visualized in a SpectraDBResultsFrame. If multiple rows are selected results are only added to the feature list rows as suggested by @robinschmid .
best,
Ansgar

### CODE REVIEW

File: src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsLocalSpectralDBSearchParameters.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package net.sf.mzmine.modules.peaklistmethods.identification.spectraldbsearch.selectedrows;
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsLocalSpectralDBSearchParameters.java]: Please add the license header

File: src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsLocalSpectralDBSearchTask.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
+package net.sf.mzmine.modules.peaklistmethods.identification.spectraldbsearch.selectedrows;
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsLocalSpectralDBSearchTask.java]: Please add the license header

File: src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsSpectralMatchTask.java [hunk 3]
```diff
@@ -0,0 +1,338 @@
+package net.sf.mzmine.modules.peaklistmethods.identification.spectraldbsearch.selectedrows;
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/spectraldbsearch/selectedrows/SelectedRowsSpectralMatchTask.java]: Please add the license header

File: src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java [hunk 4]
```diff
@@ -166,7 +168,8 @@ public PeakListTablePopupMenu(final PeakListTableWindow window, PeakListTable li
 
     searchMenu = new JMenu("Search");
     add(searchMenu);
-    dbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search online database", this);
+    onlineDbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search online database", this);
+    spectralDbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search spectral database", this);
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java]: Let's change these two menu items to "Search online compound database" and "Search local spectral database"

Comment (Annexhc) [reply src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java]: okay

**Comment by tomas-pluskal** [comment]
Looks good, thanks!

**Pull request merged**